frontend/sumo-championship/src/hooks/useNavigation.tsx

```diff
@@ -0,0 +1,21 @@
+import { useNavigate } from "react-router-dom";
+
+const useNavigation = () => {
```

Comment:
I'm not sure if this hook is necessary, we can just use `useNavigate` like it does. I think it would be better to create ROUTES.ts file and store all const variables inside it, like navigation paths. For example:

ROUTES.ts
```typescript
const ROUTES = {
   HOME: 'home',
   LOGIN: 'login',
   PLAYERS: 'players'
}
export default ROUTES;
```

After creatig it, we can just use: `navigate(ROUTES.HOME)`